hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/audit/AuditEvent.java

```diff
@@ -0,0 +1,47 @@
+package pl.allegro.tech.hermes.management.infrastructure.audit;
+
+public class AuditEvent {
+    private final AuditEventType eventType;
+    private final String resourceName;
+    private final Class<?> payloadClass;
```

Comment:
I guess there is no similar problem with this field like I described bellow, but for consistency let's change its type to `String` as well.